scripts/kubernetes/serversChart/templates/mps_webDeployment.yaml

```diff
@@ -52,6 +52,10 @@ spec:
             value: "{{ .Values.mps.enableLogging }}"
           - name: "MPS_DEBUG"
             value: "{{ .Values.mps.debug }}"
+          - name: "MPS_USE_DB"
+            value: "{{ .Values.mps.useDB }}"
+          - name: "MPS_CONNECTION_STRING"
+            value: "postgresql://{{ .Values.backenddb.dbUser }}:{{ .Values.backenddb.dbPassword }}@{{ .Values.rps.dbHost }}:{{ .Values.rps.dbPort }}/mpsdb"
```

Comment:
just double checking that for RPS -- you've updated the connection strings to reflect `/rpsdb` ?

Reply:
i haven't changed it from what it was.. the init.sql doesn't name the db like initMPS.sql does.

Reply:
Right, it's specified in the Postgres env , it should match what we have in the compose. So just making sure that's set to rpsdb, as well as connection string